exercises/concept/tisbury-treasure-hunt/.docs/instructions.md

```diff
@@ -58,7 +58,7 @@ Implement the `get_coordinate()` function that takes a `(treasure, coordinate)`
 
 
 ```python
->>> get_coordinate(("Scrimshaw Whale's Tooth", "2A"))
+>>> get_coordinate(('Scrimshawed Whale Tooth', "2A"))
```

Comment:
Not sure if this is ready for review, but I noticed the single quotes then double quotes.  It does not matter, but as a human reading the line, I wonder if there is a subtle communication/accent being given here to notice some nuance. Indeed, this seems to be a unique feature of this line specifically, where other lines seem to match quotation styles.

Reply:
This was intended for @bobahop review, which is why he was requested.  This was a copy-pasta error on my part.  These should all be single quotes.